src/i18n/en.json

```diff
@@ -45,8 +45,8 @@
     "txHash": "Transaction Hash",
     "receiveTitle": "Receive",
     "walletAddress": "Wallet Address",
-    "copyAddress": "Copy Address",
-    "copySuccess": "Copied",
+    "copyAddress": "Address Copied to Clipboard",
```

Comment:
where is this used?

Reply:
If you go to the Staking tab, open the Validators tab, click on a validator to get to the Validator Info page and then click on its address or the "Address" label.

Reply:
![image](https://user-images.githubusercontent.com/38229038/132770298-e67bcb2b-d83d-414b-a36c-9b99039656be.png)

this is from before. what's it like after the change?

Reply:
oops wrong screenshot, will update

Reply:
I don't have the screenshot but the message says: "Address Copied to Clipboard"

Reply:
Oh, nevermind, the `"copyAddress"` i18n string is actually never used.

I'll remove it.